recipes/ocetrac/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "ocetrac" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ocetrac-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d0dc96b761acef38816336c7f4c9152fcaf94de3623a47c1ddb402bc43f68c13
+  
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pytest
+    - numpy
+    - xarray
+    - dask
+    - scipy
+    - scikit-image
+
+test:
+  requires:
+    - pytest
```

We only do an import test, so we don't need `pytest` but we should check pip consistency.
```suggestion
  requires:
    - pip
  commands:
    - pip check
```

---

Thank you @xhochy! I've made your changes.